plugin/pkg/scheduler/algorithm/priorities/test_util.go

```diff
@@ -41,18 +41,18 @@ func makeNode(node string, milliCPU, memory int64) *v1.Node {
 	}
 }
 
-func priorityFunction(mapFn algorithm.PriorityMapFunction, reduceFn algorithm.PriorityReduceFunction) algorithm.PriorityFunction {
+func priorityFunction(mataData interface{}, mapFn algorithm.PriorityMapFunction, reduceFn algorithm.PriorityReduceFunction) algorithm.PriorityFunction {
```

Comment:
feel free to ignore it. first parameter being nil most of the time looks weird. move it to the end?

Reply:
reasonable, thanks.